paasta_tools/vitesscluster_tools.py

```diff
@@ -72,21 +662,98 @@ def validate(
             return []
 
 
-def load_vitess_instance_config(
+def generate_vitess_instance_config(
+    instance_config: Dict,
+) -> Dict:
+    # Generate the vitess instance config from the yelpsoa config
+
+    cpus = instance_config.get("cpus")
+    mem = instance_config.get("mem")
+    deploy_group = instance_config.get("deploy_group")
+    zk_address = instance_config.get("zk_address")
+    paasta_pool = instance_config.get("paasta_pool")
+    paasta_cluster = instance_config.get("paasta_cluster")
+    cells = instance_config.get("cells")
+    keyspaces = instance_config.get("keyspaces")
+    region = instance_config.get("region")
+
+    vitess_instance_config = {
+        "namespace": "paasta-vitessclusters",
+        "cpus": cpus,
+        "mem": mem,
+        "min_instances": 1,
+        "max_instances": 1,
+        "deploy_group": deploy_group,
+        "autoscaling": {"setpoint": 0.7},
+        "env": {
+            "OPERATOR_NAME": "vitess-operator",
+            "POD_NAME": "vitess-k8s",
+            "PS_OPERATOR_POD_NAME": "vitess-k8s",
+            "PS_OPERATOR_POD_NAMESPACE": "paasta-vitessclusters",
+            "WATCH_NAMESPACE": "paasta-vitessclusters",
+        },
+        "healthcheck_grace_period_seconds": 60,
```

Comment:
might be good to make this configurable from the start - iirc, there was an incident for one of the other operator owners a while back where their healthchecks slowed down for some reason and it was a bit of a pain to make the healthcheck config configurable (or edit it to start with) during the incident)